src/controllers/browser.ts

```diff
@@ -434,3 +435,15 @@ function isChromeInstalled(executablePath: string): boolean {
     return false;
   }
 }
+
+function removeStoredSingletonLock(puppeteerOptions): boolean {
+  const platform = os.platform();
+  const { userDataDir } = puppeteerOptions;
+  try {
+    if (platform === 'win32') return false;
+    fs.unlinkSync(`${userDataDir}/SingletonLock`);
```

Comment:
This path is not for all users, that is, it is not global!

Reply:
I will check.

Reply:
If I delete the SingletonLock folder, will the login work normally? In other words, will it be necessary to scan the QR code again?